js/src/lib/interfaces/Libraries.ts

```diff
@@ -158,12 +158,17 @@ model = from_pretrained_keras("${model.id}")
 `;
 
 const paddlenlp = (model: ModelData) => {
-	return [
-	  `from paddlenlp.transformers import AutoModel, AutoTokenizer`,
-	  "",
-	  `tokenizer = AutoTokenizer.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
-	  `model = AutoModel.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
-	].join("\n");
+	if (model.config && model.config.architectures && model.config.architectures[0]) {
```

Comment:
I think it's also possible to do

```suggestion
	if (model.config?.architectures?.[0]) {
```